package.json

```diff
@@ -12,6 +12,7 @@
     "@antv/g2": "^4.1.30",
     "ant-design-vue": "^1.7.2",
     "axios": "^0.21.2",
+    "big-integer": "^1.6.52",
```

Comment:
Discussed in QQ Group: it's fine to add `big-integer` deps here. In the future, when we targeting at modern web platform, use native BigInt instead would be great.